package.json

```diff
@@ -14,38 +14,20 @@
     "check-pretty": "prettier --check .",
     "lint": "solhint \"contracts/**/*.sol\"",
     "compile": "hardhat compile --show-stack-traces --force && forge build --force",
-    "test-hardhat": "hardhat test",
-    "test-foundry-local": "forge test -vvv --force --no-match-path 'contracts/test/forkMainnet/*.t.sol'",
-    "test-foundry-fork": "forge test -vvv --force --fork-url $MAINNET_NODE_RPC_URL --fork-block-number $FORK_BLOCK_NUMBER --match-path 'contracts/test/forkMainnet/*.t.sol'",
-    "test-foundry-local-no-fuzz": "forge test -vvv --force --no-match-path 'contracts/test/forkMainnet/*.t.sol' --no-match-test testFuzz_ConvertVeLontoMigrateStake"
+    "test-foundry-local": "DEPLOYED=false forge test -vvv --force --no-match-path 'contracts/test/forkMainnet/*.t.sol'",
+    "test-foundry-fork": "DEPLOYED=false forge test -vvv --force --fork-url $MAINNET_NODE_RPC_URL --fork-block-number 15451000 --match-path 'contracts/test/forkMainnet/*.t.sol'",
```

Comment:
Do we need to replace `FORK_BLOCK_NUMBER` with hardcoded block number?

Reply:
Actually it's applying changes in `master`. I think it can be more consistent when running tests in local and CI as well.